### PR TITLE
How to Contribute tutorial

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: docs/how2contribute.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+
+
+
+Files
+############
+The source/filler test file is written in either YML or JSON and located
+under the **src** directory. In addition to this file, your pull request
+needs to include the generated/filled version(s) of the test file.
+
```
Comment (winsvega) [inline docs/how2contribute.rst:64]: Can add more description about what is the difference of source from filled test

File: docs/how2contribute.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+State tests have their source at 
+**src/GeneralStateTestsFiller/<directory>/<test>Filler.<yml or json>**. 
+Every state test is supposed to have two filled versions:
+
+#. Filled state test, which is located in **GeneralStateTests/<directory>/<test>.json**.
+   You use a command similar to this one to create this file:
+
+   ::
+
+      ./dretesteth.sh -t $suite -- --testpath $dir --singletest $test --filltests
```
Comment (winsvega) [inline docs/how2contribute.rst:77]: Add a link here about how to setup dretesteth

**Review by winsvega — COMMENTED** [review]
Put comments

**Comment by qbzzt** [comment]
Sorry, I will do this over the weekend. Life's been hectic.

**Pull request merged**